lib/runtime/evaluation.coffee

```diff
@@ -28,6 +26,12 @@ module.exports =
           @showError r, result.highlights
         notifications.show "Evaluation Finished"
 
+  evalCode: (text, line=1) ->
```

Comment:
Yea it was just the most obvious way to do it at the time. I didn't understand how to do an rpc msg